myscript.js

```diff
@@ -0,0 +1,198 @@
+
+const myUrl =  "http://localhost:5050/getFibonacciResults";
+let fibonacciUrl = "http://localhost:5050/fibonacci/";
+
+let myArr = [];
```

Comment:
This should not be so global. You should create it inside the functions that you use it.